schema-lending.graphql

```diff
@@ -332,9 +256,18 @@ type Market @entity {
   " Total borrow volume in USD (i.e. total loan origination ) "
   totalVolumeUSD: BigDecimal!
 
+  " Current balance of all deposited assets (not historical cumulative), in USD "
+  totalDepositUSD: BigDecimal!
+
+  " Current balance of all borrowed assets (not historical cumulative), in USD. In the case of CDPs, this will be all minted assets "
+  totalBorrowUSD: BigDecimal!
+
   " Amount of input tokens in the market. The ordering should be the same as the market's `inputTokens` field. "
   inputTokenBalances: [BigInt!]!
```

Comment:
I think it could be useful to specify more on what this exactly is. i had it wrong at first

Reply:
hmm, can you give me an example of how I can clarify this?